thor/src/lib.rs

```diff
@@ -41,7 +41,7 @@ use crate::{
 use ::serde::{Deserialize, Serialize};
 use anyhow::{anyhow, bail, Result};
 use async_trait::async_trait;
-use bitcoin::{Address, Amount, Transaction, Txid};
+use bitcoin::{util::amount::serde::as_sat, Address, Amount, Transaction, Txid};
```

Comment:
You need to put this import behind `feature = "serde"` or clippy will fail if someone compiles without the feature.

I setup the CI to build with and without the feature but not clippy.

Reply:
Cool, done. Thanks.